aspnetcore/blazor/components/index.md

```diff
@@ -1357,45 +1357,16 @@ Attributes can be applied to components with the [`@attribute`][7] directive. Th
 @attribute [Authorize]
 ```
 
-## Conditional HTML element attributes
+## Conditional HTML element attributes and DOM properties
 
-HTML element attribute properties are conditionally set based on the .NET value. If the value is `false` or `null`, the property isn't set. If the value is `true`, the property is set.
+The distinction between HTML element attributes and DOM properties is subtle, and complicated rules determine how attributes and properties are related and updated. In some cases, updating an attribute updates the property automatically, while in other cases it doesn't.
```

Comment:
I don't personally think it's helpful to make this remark about it being subtle/complicated. It's true that it's complicated, but JS/DOM is an external system that we don't have to make value judgements about. People can have their own opinion about this and we neither need to justify nor document it.

The other content below looks fine though, and makes just as much sense without this statement about complexity.